CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Fixed
 - Allow CSS property values containing newlines
   ([#504](https://github.com/MyIntervals/emogrifier/pull/504))
+- Allow adjacent sibling CSS selector combinator in minified CSS
+  ([#505](https://github.com/MyIntervals/emogrifier/pull/505))
```

Comment:
The change log entries should be in reverse chronological oder (i.e., newest on top). I'll change that in a minute.